src/Sylius/Component/Core/composer.json

```diff
@@ -27,6 +27,7 @@
     ],
     "require": {
         "php": "^8.2",
+        "doctrine/common": "^3.0.3",
```

Comment:
doctrine/orm 2.18.0 was required doctrine/common 3.0.3, so I choose this version.
https://packagist.org/packages/doctrine/orm#2.18.0